package.json

```diff
@@ -55,28 +66,23 @@
 		"buffer": "^6.0.3",
 		"bufferutil": "^4.0.6",
 		"crypto-browserify": "^3.12.0",
+		"cypress": "^9.6.0",
+		"cypress-jest-adapter": "^0.1.1",
```

Comment:
My concern with using this helper package is that it may not be actively maintained

![image](https://user-images.githubusercontent.com/20375610/165791867-686de9f8-91dc-41cc-98bc-30ab498e75b0.png)

So if we plan to use it, we may need to maintain it ourselves. Even the dependencies haven't been upgraded in over a year:

![image](https://user-images.githubusercontent.com/20375610/165792086-4ea49038-19f8-42c5-9cd6-166ed8856845.png)

Reply:
Yes it's not been updated because the the Jest API is not updated since long time and it's just the DSL wrapper on top of it.

Reply:
For dependencies, we may fork and update those. But since it's just the development dependency, I believe it's fine to use.

Reply:
Created an [issue](https://github.com/ChainSafe/web3.js/issues/5002) to track this